PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/azure-intellij-plugin-azure-sdk.xml

```diff
@@ -0,0 +1,38 @@
+<idea-plugin>
+    <depends>org.jetbrains.idea.maven</depends>
+    <depends>com.intellij.modules.xml</depends>
+    <depends>com.intellij.modules.java</depends>
+
+    <actions>
+            <action
+                    class="com.microsoft.azure.toolkit.intellij.azure.sdk.experiment.ExperimentalFeature"
+                    id="AzSdkHelpAction"
+                    text="Ask Azure SDK Help">
+                <add-to-group group-id="ToolsMenu" anchor="last" />
+            </action>
+    </actions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <localInspection
+                language="XML"
+                shortName="MavenProjectInspection"
+                displayName="Check Maven Project"
+                groupName="Maven"
+                enabledByDefault="true"
+                level="WARNING"
+                implementationClass="com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.MavenProjectInspection">
+            <class>com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.MavenProjectInspection</class>
+        </localInspection>
+
+        <localInspection
+            language="JAVA"
+            shortName="UseStorageAPIsWithLengthParameter"
+            displayName="Use Storage APIs with Length Parameter"
+            groupName="Azure SDK Rules"
+            enabledByDefault="true"
+            level="WARNING"
```

Comment:
May want the level for this to be whatever is slightly less than a warning. Using the APIs are valid and generally safe but hide a lot of complexity and control. So, I don't think warning about their usage is the correct level but informationally telling users that if finer control is needed it may be better to use the more manual APIs that take lengths or require managing their data before uploading.

Reply:
changed to INFO